ariel/get_ec2_pricing.py

```diff
@@ -149,8 +149,27 @@ def load(config, locations=LOCATIONS):
         return prices
 
 def check_row(header_map, row):
-
-    if row[header_map['Product Family']] != 'Compute Instance':
+    
+    if row[header_map['Product Family']] != 'Compute Instance' and \
```

Comment:
The pythonic way of doing this is `row[header_map['Product Family']] not in ['Compute Instance', 'Compute Instance (bare metal)`.  Just an FYI, it doesn't matter either way.